cmd/tesla-control/commands.go

```diff
@@ -517,7 +517,7 @@ var commands = map[string]*Command{
 	},
 	"wake": &Command{
 		help:             "Wake up vehicle",
-		requiresAuth:     false,
+		requiresAuth:     true,
```

Comment:
The issue with this change is that the Wakeup command will now fail if it goes over the Internet and the session is not cached. In this case, the client will try to handshake with the sleeping car before sending the wake command.

The Command structure currently has no way of indicating that a private key is conditionally required. Since "wake" will almost certainly remain the unique case that requires conditional logic, one option would be to add special handling for wake in the `configureFlags` function. E.g., (this is not tested):

```go
    if info.requiresAuth || (forceBLE && commandName == "wake") {                                                                                                                    
        c.Flags |= cli.FlagPrivateKey | cli.FlagVIN                                                                                           
    } 
```